colibri/Cargo.toml

```diff
@@ -9,7 +9,7 @@ log = "0.4.22"
 simplelog = "0.12.2"
 # For the asynchronous Rest API
 reqwest = { version = "0.12.2", features = ["json"]}
-tokio = { version = "1.43.0", features = ["rt-multi-thread", "fs"]}
+tokio = { version = "1.43.0", features = ["rt-multi-thread", "fs", "test-util"]}
```

Comment:
what is test-util tokio feature for? We probably need some more comments here ot be understanding why we have stuff.

Reply:
This one is for using tokio in tests, tests that use the async runtime. they use `#[tokio::test]`